### ISSUE TITLE
Failing Behat test - filter_embedquestion.feature

### ISSUE THREAD

**Issue by AdilIqbal95** [moodleou/moodle-filter_embedquestion#20]
Hi, the test stated above is failing.

**Environment details:**
MySQL 5.7
PHP 7.4
Moodle latest MOODLE_400_STABLE

**Error details:**
**Scenario: Test using the helper script - embed a specific question**
And I reload the page: Dismissed user prompt dialog: To display this page, Firefox must send information that will repeat any action (such as a search or order confirmation) that was performed earlier. (Facebook\WebDriver\Exception\UnexpectedAlertOpenException)

**Scenario: Links to and from the question bank**
Then I should see "Editing a True/False question": "Editing a True/False question" text was not found in the page (Behat\Mink\Exception\ExpectationException)

**Comment by sharpchi** [comment]
Yes I get the same error.
MariaDB: 10.3
PHP 8.0
M 4.0.4

Wonder if it's something to do with the browser being used by Selenium: Firefox, in my case.

**Comment by sharpchi** [comment]
Yes, it seems so. For a reason I can't figure out right now, I can't run the Chrome selenium browser, so Firefox is the only one that's currently working for me. I tried using the latest versions of Firefox and Chrome, and no change.
It's not the end of the world as long as it's working somewhere! I'm just reviewing the plugins we use for M4, and I'm sure you're on top of this one.
